### ISSUE TITLE
论文问题

### ISSUE THREAD

**Issue by XXD-N** [CUHK-AIM-Group/DiffRect#3]
哥们你论文写的乱的，我以为我自己智商除了b问题，你是碰到熟人了吧给你中了

**Comment by xinyuliu-jeffrey** [comment]
Hi @XXD-N ,

Thank you for your interest in our work. If you have questions about the article, please feel free to raise them here instead of making meaningless outbursts. Additionally, the reviewers' comments are publicly accessible on the MICCAI website, so you're welcome to check them.

**Comment by XXD-N** [comment]
为我的一时冲动道歉，我理解你论文的目的，但是你的论文组织很有问题，我这里有和一个对你们文章表示弱拒绝的一个审稿人一样的疑问，你们文章的输入和输出很不明确，而且你的流程图十分不清晰，代码论述中也存在表述不清楚的问题：
![aa](https://github.com/user-attachments/assets/f8ab692d-ed44-4e3e-b297-3df8a127fc0c)
![bb](https://github.com/user-attachments/assets/cee75891-6107-4c3f-ab96-b75796979178)

**Comment by XXD-N** [comment]
Apologize for my impulsive remarks. I understand the purpose of your paper, but there are problems with the organization of your paper. I have the same questions as a reviewer who expressed a weak rejection of your article. The input and output of your article are very unclear, and your flow chart It is very unclear, and the explanation of the paper is also very unclear:
![aa](https://github.com/user-attachments/assets/e0e0909c-65bb-471c-9e76-cb148a6574c1)
![bb](https://github.com/user-attachments/assets/2b8bdbe0-d3a7-4bc9-9335-6324e2482cb5)

**Comment by xinyuliu-jeffrey** [comment]
Hi @XXD-N ,

Thank you for the detailed question and sorry for the unclarity of the paper. 

The main reason is due to the page limitation of the MICCAI, so we indeed omitted the baseline segmentation part (which is the (c) in the main architecture here) and spent more pages in our proposed components (a) and (b), which makes the overall pipeline unclear for those who are not familiar with the problem definition. 

Our segmentation model baseline is the Fixmatch (https://[arxiv.org/pdf/2001.07685](https://arxiv.org/pdf/2001.07685)), which feeds the model with weakly augmentated data and strongly augmented data during training. The strongly augmentated data generates poor pseudo labels and weakly augmented data generates good ones. The semi loss learns a consistency between their output pseudo labels to train the segmentation model \theta. 

Our LFR module learns to refine pseudo labels as we have poor and good pseudo label pairs. So after learning, LFR can further improve the good pseudo-label of the weakly-augmented data to generate a "better" pseudo label (which is r_l in the figure), and we use the r_l to provide a better supervision to the segmentation model \theta in (c). 

In conclusion, our framework basically starts at I_s and I_w in (c), and we train the segmentation unet with both the consistency loss in Fixmatch and the supervision loss with the pseudo label r_l. Therefore, the model could improve its segmentation results. During inference, (a) and (b) are not used.

I hope this clarifies a little bit. As we also released our code with detailed comments, you may also check the code directly, which could be more helpful. If there is anywhere confusing, feel free to ask and I'll try my best to clarify.
![image](https://github.com/user-attachments/assets/d3a09af8-9e6d-4ef1-9aac-c3b7e1a795d2)

**Comment by xinyuliu-jeffrey** [comment]
Closed due to inactivity. Feel free to reopen.
